source/connect.txt

```diff
@@ -204,6 +204,23 @@ For ``tls`` connections:
 
         mongosh "mongodb://mongodb0.example.com:28015" --tls
 
+Connect to a Specific Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a specific database, specify a database in your
+`connection string URI path
+<https://docs.mongodb.com/manual/reference/connection-string/>`__. If
+you do not specify a database in your URI path, you connect to a Test
+database by default.
+
+   .. example::
+
+     The following connection string URI connects to database db1.
```

Comment:
[nit] Let's also use the double backticks around `db1` to make it standout a bit more.

```
The following connection string URI connects to database ``db1``.
```